Library/Homebrew/test/rubocops/patches_cop_spec.rb

```diff
@@ -108,6 +109,15 @@ def patches
                                   line: 5,
                                   column: 5,
                                   source: source }]
+        elsif patch_url =~ %r{https?://github\.com/.+/.+/(?:commit|pull)/[a-fA-F0-9]*.(?:patch|diff)}
+          expected_offenses = [{  message: <<-EOS.undent,
```

Comment:
Any reason why there are two spaces here after the bracket instead of one?

Reply:
No. Will make it one.

Reply:
Also, it's weird that this isn't reported by RuboCop, since it usually suggests moving the variable outside of the `if`, when it is the same assignment in all branches.

Reply:
https://github.com/Homebrew/brew/blob/master/Library/.rubocop.yml#L111

Reply:
I think this should be enabled for core code.

Reply:
@reitermarkus @GauthamGoli I'm 🆒 with PRs to make changes like that. I'd like to see us get closer to the RuboCop defaults on as much as possible.